day_1/parse_tag.rb

```diff
@@ -0,0 +1,60 @@
+#take in a string from an html file
+#parse into hash or struct (regex)
+#likely multiple regex's 
+
+ParsedStringStruct = Struct.new(:type,:classes,:id,:name,:src,:title)
+
+class ParseTag
+
+  def parse_tag(string)
+    
+
+    parsed_string = ParsedStringStruct.new
+
+    #type
+    regex = /<([a-zA-Z]+)/
+    type = string.match(regex)
+    parsed_string.type = type[1] if type
+
+    #class
+    regex =/class='(.*?)'/
```

Comment:
Do you notice a pattern in lines 20-37?

Perhaps it would be possible to extract the duplication into a reusable method?